README.md

```diff
@@ -80,6 +80,7 @@
 
 ## dApps
 
+- [NEO dApp List (neo.org)](https://neo.org/dapps)
```

Comment:
Hi there! Please provide a short description of the link you add, like the link below it :)